pkg/proxy/ipvs/proxier.go

```diff
@@ -1861,22 +1822,3 @@ func openLocalPort(lp *utilproxy.LocalPort) (utilproxy.Closeable, error) {
 	glog.V(2).Infof("Opened local port %s", lp.String())
 	return socket, nil
 }
-
-// ipvs Proxier fall back on iptables when it needs to do SNAT for engress packets
```

Comment:
Please keep the comment until we have better document.

Reply:
I see, thanks.